src/ex2/itemManager.js

```diff
@@ -0,0 +1,26 @@
+export class ItemManager {
```

Comment:
in terms of resposibilities it makes more sense that the item manager will be in charge of the whole pokemon API requests and logic, app.js should only care for the UI itself, item manager should contain all of your non-ui logic